eng/Tools.props

```diff
@@ -1,11 +1,11 @@
 <Project>
   <ItemGroup>
-    <PackageReference Include="MicroBuild.Core" Version="$(MicroBuildCoreVersion)"  />
+    <!-- <PackageReference Include="MicroBuild.Core" Version="$(MicroBuildCoreVersion)"  />
```

Comment:
Please either remove or add a comment why this is commented out.

Reply:
It's part of clean-up and will be deleted.